tests/merge_ntfs.rs

```diff
@@ -23,20 +23,22 @@ use transit_model::objects::{Comment, CommentLinks, StopPoint, VehicleJourney};
 use transit_model::test_utils::*;
 use transit_model::transfers;
 use transit_model::transfers::TransfersMode;
-use transit_model_collection::CollectionWithId;
-use transit_model_collection::Idx;
+use typed_index_collection::CollectionWithId;
+use typed_index_collection::Idx;
```

Comment:
Maybe they can be merged?
```suggestion
use typed_index_collection::{CollectionWithId, Idx};
```

Reply:
maybe :)